Content.Server/Sound/EmitSoundSystem.cs

```diff
@@ -0,0 +1,86 @@
+using Content.Shared.Audio;
+using Content.Shared.Interaction;
+using Content.Shared.Throwing;
+using Content.Server.Interaction.Components;
+using Content.Server.Sound;
+using Content.Server.Throwing;
+using JetBrains.Annotations;
+using Robust.Shared.Audio;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Player;
+using Robust.Shared.Random;
+using Robust.Shared.Log;
+
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    [UsedImplicitly]
+    public class EmitSoundSystem : EntitySystem
+    {
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [Dependency] private readonly IRobustRandom _random = default!;
+
+        /// <inheritdoc />
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<EmitSoundOnLandComponent, LandEvent>((eUI, comp, arg) => PlaySound(comp));
+            SubscribeLocalEvent<EmitSoundOnUseComponent, UseInHandEvent>((eUI, comp, arg) => PlaySound(comp));
+            SubscribeLocalEvent<EmitSoundOnThrowComponent, ThrownEvent>((eUI, comp, arg) => PlaySound(comp));
+            SubscribeLocalEvent<EmitSoundOnActivateComponent, ActivateInWorldEvent>((eUI, comp, args) => PlaySound(comp));
+        }
+
+        private void PlaySound(BaseEmitSoundComponent component)
+        {
+            if (!string.IsNullOrWhiteSpace(component._soundCollectionName))
+            {
+                PlayRandomSoundFromCollection(component);
+            }
+            else if(!string.IsNullOrWhiteSpace(component._soundName))
```

Comment:
```suggestion
            else if (!string.IsNullOrWhiteSpace(component._soundName))
```

To keep it consistent